web/app.js

```diff
@@ -450,6 +451,10 @@ const PDFViewerApplication = {
           )
         : null;
     }
+    const toastManager = new ToastManager(
+      appConfig.annotationRemovedToast,
+      10000
+    );
```

Comment:
```js
    const toastManager = new ToastManager({
      element: appConfig.annotationRemovedToast,
      timeout: 10000
    });
```

some API like this would make it easier to understand what "10000" means when reading the code.

Reply:
Has this value been specified ?

Reply:
This will break in e.g. the GeckoView viewer, note how other definitions handle this via `appConfig`-checks in this code.

Reply:
Thanks for the suggestions and to be entirely fair I had no clue what was a good thing to do in this specific case so I decided to go with something mostly arbitrary and ask you all for guidance whether the project's guidelines have something to say on this. Thanks for the pointer @Snuffleupagus, I'll look around the codebase for what to do here.

Reply:
@Snuffleupagus after a quick look, I could find most current calls to `setInterval` in the codebase atm either with hardcoded values or a literal constant defined in the respective file, could you point me to the part of the codebase you're specifically referring to above? Thanks

Reply:
*(I don't know what setInterval has to do with this, since that's not related to the issue I mentioned above.)*

Following the `viewer.js` changes, this is safe if implemented as:
```suggestion
    const editorUndoBar = appConfig.editorUndoBar
      ? new EditorUndoBar(appConfig.editorUndoBar, /* duration = */ 10000)
      : null;
```